bloom-housing/bloom
# feat: creating users as juris admin

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2763)


## Description
This handles the case of creating users as a jurisdictional admin

## How Can This Be Tested/Reviewed?
log in as a jurisdictional admin

navigate to the user grid, and move to create a user

first create another jurisdictional admin, notice that the fields match up with the mock and that the save happens successfully (the new juris admin has the same jurisdiction as the creating jurisdictional admin)

move to create a new parter, notice that the fields match up with the mock, and they are only allowed to pick listings for the jurisdiction they are assigned to 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
